scrapers/sis_scraper/main.py

```diff
@@ -329,7 +333,17 @@ async def main():
     async with aiohttp.ClientSession(
         connector=aiohttp.TCPConnector(limit=5)
     ) as session:
-        for semester in util.get_semesters_to_scrape():
+        semesters = util.get_semesters_to_scrape()
```

Comment:
would just be nice to have a usage string, other than that the changes to scraper look fine to me